src/rate/rate.js

```diff
@@ -1,3 +1,21 @@
-export function formatRate(rate) {
-  return rate.toFixed(5);
+export function formatRate(exchangeRate, sourceCurrency, targetCurrency) {
+  if (!sourceCurrency || !targetCurrency) {
+    return defaultFormatRate(exchangeRate);
+  }
+
+  return isAWeakCurrency(sourceCurrency)
+    ? formatWeakCurrencyRate(exchangeRate, sourceCurrency, targetCurrency)
+    : defaultFormatRate(exchangeRate);
+}
+
+function defaultFormatRate(exchangeRate) {
```

Comment:
Could we rename this to be more descriptive (than just default)? F.e.
```suggestion
function formatRateToDecimalPoints(exchangeRate) {
```